### PR TITLE
(PC-7331) App native - POC investigation librairie calcul de distance geoloc

### PR DESCRIPTION
**Pull request by kopax-polyconseil**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-7331

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/cheatcodes/pages/Navigation.tsx [hunk 1]
```diff
@@ -224,6 +228,19 @@ export function Navigation(): JSX.Element {
...
         </Row>
+        <Row half>
+          <NavigationButton
+            title={`Distance to Eiffel Tower`}
+            onPress={() => {
+              if (position) {
+                const distance = getDistance(position, eiffelTowerCoordinates)
+                Alert.alert(`${distance / 1000}km`)
+              } else {
+                Alert.alert('Authorize geolocation first')
```
Comment (BenStirrup) [inline src/features/cheatcodes/pages/Navigation.tsx:239]: Juste une note sur le message : on peut ne pas avoir la position pour d'autres raisons que la permission.

**Review by BenStirrup — APPROVED** [review]

File: src/features/cheatcodes/pages/Navigation.tsx (hunk 1 above)
Comment (kopax-polyconseil) [reply src/features/cheatcodes/pages/Navigation.tsx:239]: Tu as des exemples ? La puce est accessible offline.

Comment (BenStirrup) [reply src/features/cheatcodes/pages/Navigation.tsx:239]: Parfois ca peut fail et le position n'est pas dispo. @blixit s'y connait dessus si besoin de plus d'info

Comment (kopax-polyconseil) [reply src/features/cheatcodes/pages/Navigation.tsx:239]: OK et vue avec @blixit

**Pull request merged**